.github/workflows/coverage-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Coverage
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+
+  build-and-test:
+    name: coverage
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: install branchpro
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .
+          python -m pip install coverage codecov
+      - name: run coverage
+        run: |
+          coverage run run-tests.py --unit
+
+      - name: codecov
+        if: success()
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I don't know for sure, but this might not be necessary, codecov says "Token not required" for Github actions. 
Let's leave it for now as it seems to work fine.